### DISCUSSION TITLE
new Ezcaps on market ?

### DISCUSSION THREAD

**Discussion by el-bakkali** [awawa-dev/HyperHDR#954] [Q&A]
Hi all
im looking to upgrade my capture card from MS2109,
im looming at the available support downloadable Lut and I can see only a FEW ezcap models
while searching aliexpress I noticed three capture cards and wanted to know if they are recommend
They are the ezcap333 and the ezcap371 and Ezcap326

any thoughts ? would these be ok for hyperhdr  ?

**Comment by awawa-dev** [comment]
Hi
These are different times... if you have USB3.0, choose new MS2130 for $10, if you want high-end and not worry about whether the USB-C socket will start falling off & firmware issues and locked color settings (my Ezcap320) or overheating (because it is made of cheap plastic: my ezcap 331... even generic MS2109 or MS2130 for $10 have metal casings), choose Elgato for example HD60X. For HDCP or Dolby Vision (LLDV) choose the Ezcoo splitter. And if you have USB2.0 only, you can do nothing and stay with MS2109.

**Reply by el-bakkali** [reply]
thanks for guidance
I need one that has splitter 
would the ezcap 321 be ok ?
https://www.aliexpress.com/item/1005005931798448.html

Or do you have any recommendation for MS2130 in similar form factor

**Reply by awawa-dev** [reply]
Built-in splitters (grabber + splitter in one) are usually big problems: if the grabber does not support HDR, it will not pass the HDR signal (although if it is forced by the splitter, it will catch it and HyperHDR will fix the colors), it will block recording if it detects HDCP, it may have limited EDID and e.g. not offer 1080p24 signal even though both TV and grabber support it... if you do not know the answers to these questions about a given grabber with a loop, it is better to forget about it. Usually these parameters are not given in brochures and only tests can tell something if you know what you are looking for.
The only such trouble-free from the beginning grabber+splitter that I have is the Elgato HD60X, but even it does not support HDCP due to licensing restrictions.

**Reply by el-bakkali** [reply]
thank you

I think i have my answer then for my budget
MS2130 with Ezcoo splitter
